### DISCUSSION TITLE
Cannot set Production Branch for a Preview Domain??

### DISCUSSION THREAD

**Discussion by maxiedaniels** [vercel/community#4624] [Help]
### Summary

I’m getting this error when I try and set my domain to my production branch, but I can’t find any indication in domain settings about “preview domain” for my domain. I do see that my domain says “production” on the main domains list, so I’m not sure what’s happening. Could this be a bug??

### Steps to Reproduce

Not much to explain

**Comment by swarnava** [comment]
Hi, Can you share a screenshot?
